packages/universal-components/src/Touchable/Touchable.js

```diff
@@ -103,28 +103,27 @@ export default class Touchable extends React.Component<Props> {
       }
 
       return (
-        <TouchableNativeFeedback
-          {...touchableNativeFeedbackProps}
-          style={{ width }}
-          useForeground={useForeground}
-          background={TouchableNativeFeedback.Ripple(
-            rippleColor,
-            borderlessRipple,
-          )}
-        >
-          <View style={style}>{children}</View>
-        </TouchableNativeFeedback>
+        <View style={{ width }}>
```

Comment:
Do we need this extra wrapper? 
I think that updating View inside touchable can be enough.
`<View style={[style, { width }]}>{children}</View>`

Reply:
True :) done